download_sdl2_hg.cmake

```diff
@@ -26,11 +26,12 @@ endif()
 ExternalProject_Add(
     SDL2HG
     PREFIX ${CMAKE_BINARY_DIR}/external/SDL2
-    URL https://hg.libsdl.org/SDL/archive/default.tar.bz2
+    URL https://www.libsdl.org/release/SDL2-2.0.8.tar.gz
```

Comment:
Why not to make the option:
```cmake
option(SDL_USE_RELEASE ...)
if(SDL_USE_RELEASE)
    set(SDL_TAR_URL "https://www.libsdl.org/release/SDL2-2.0.8.tar.gz")
else()
    set(SDL_TAR_URL  "https://hg.libsdl.org/SDL/archive/default.tar.bz2")
endif()
```
It's better to give a choice than force one of...